docs/src/index.md

```diff
@@ -0,0 +1,23 @@
+# Blink.jl Documentation
+
+## Overview
+
+Blink.jl provides a API for communicating with web pages from Julia. Pages may
+be served over the internet and controlled from the browser, or served locally
+via an [Electron](https://electronjs.org/) window. Blink can therefore be used
+as a GUI toolkit – DevTools.jl for an example use.
```

Comment:
I copied this from the README, but it seems a bit out of date. Does anyone use Blink as the backend of a web server anymore? this seems confusing to include in the main introduction to the package.

I would probably describe it instead as something like:

"Blink.jl is the Julia wrapper around Electron. It can serve HTML content in a local window, and allows for communication between Julia and the web page. In this way, therefore, Blink can be used
as a GUI toolkit for building HTML-based applications for the desktop."

What do you think?

Reply:
Yep, that's better, imho.

Reply:
:) Thanks! Done.